src/base/bittorrent/session.cpp

```diff
@@ -141,19 +141,55 @@ namespace
         return true;
     }
 
+#ifdef Q_OS_WIN
+    // Remove this after 4.2.5 && if at least one month has passed from v4.2.3
+    // Check the commit that introduced this function and identify all other pieces of code that
+    // need removal alongside this one.
+    void fixBrokenSavePath(QByteArray &data, CreateTorrentParams &torrentParams, lt::bdecode_node &root)
+    {
+        const QString path = fromLTString(root.dict_find_string_value("save_path"));
+        const int index = path.indexOf(QLatin1String("//"));
```

Comment:
@Tester798 in your `go` code I see  that you also replace `/` with `\` in string. Is that necessary?

On another note, is it possible for the broken paths to appear with `\\` instead of `//`?

Reply:
Looks like it doesn't work if symbols `/` aren't replaced with `\`:
![image](https://user-images.githubusercontent.com/8844478/77836273-68100900-715d-11ea-9af2-2310686731f3.png)

If I add `replace` like this
`const QString goodPath = path.mid(index).replace('/', '\\');`
then it works fine.

Reply:
I wasn't able to get broken paths with `\\` in them instead of `//`, but I also wasn't able to get paths like `C:\Program Files\qBittorrent\//desktop-1234567/downloads/` too (only by manually editing fastresume files), so I'm not really sure what is possible. Probably we should check ppl reports, and I didn't see reports with paths like `C:\Program Files\qBittorrent\\\desktop-1234567\downloads\` yet, so probably it isn't possible to get them.